### ISSUE TITLE
List appropriate inventory on beer page

### ISSUE THREAD

**Issue by alefort** [alefort/getcrafty#18]
Right now we're listing the LCBO province wide inventory, we should be showing the inventory at the specific store we selected.

I noticed that the url has two parameters, is the second the store id? Is this a rough in to support this inventory data? Like

http://www.getcrafty.co/app/public/#/beer/16329/393

Is the 393 in the url the store id?

**Comment by scottblinch** [comment]
Yep, had to slap the store ID on there in order to make the select a new beer buttons work.

For the following beer/store combination:
http://www.getcrafty.co/app/public/#/beer/16329/438

I'm trying to use this API call in order to get inventory information for that specific store:
http://www.getcrafty.co:3000/api/v1/productsAtStore?store_id=438&beer_id=16329

But it's returning beer ID 6668 (Staropramen). Am I coming at this the wrong way?

**Comment by scottblinch** [comment]
Actually, hang on, I have a feeling I'm being dumb.

**Comment by scottblinch** [comment]
Okay, I think I'm simply using that API call incorrectly. It just returns all the store's beers rather than one I select. Do we have the ability to just get the inventory information for a specific beer at a specific store, or should I just be grabbing all the store's beers on the beer page?

**Comment by alefort** [comment]
We have an api call for this. Will send over tonight.

**Comment by alefort** [comment]
Scott, you were close. There is a REST model for INVENTORIES you can use, and the parameters are store_id and product_id (you had beer_id).

http://www.getcrafty.co:3000/api/v1/inventories?store_id=438&product_id=16329
